signingscript/script.py

```diff
@@ -136,15 +138,11 @@ def main(config_path=None):
         sslcontext = ssl.create_default_context(cafile=context.config['ssl_cert'])
         kwargs['ssl_context'] = sslcontext
     conn = aiohttp.TCPConnector(**kwargs)
-    with aiohttp.ClientSession(connector=conn) as session:
-        context.session = session
-        try:
-            loop.run_until_complete(async_main(context))
-        except ScriptWorkerTaskException as exc:
-            traceback.print_exc()
-            sys.exit(exc.exit_code)
-        finally:
-            loop.close()
```

Comment:
I just ran signingscript locally, and didn't get any unclosed loop errors... yay!